resources/sdk/clisdkclient/templates/root.mustache

```diff
@@ -35,9 +48,52 @@ var versionCmd = &cobra.Command{
 	Long:  `All software has versions. This is gc version's`,
 	Run: func(cmd *cobra.Command, args []string) {
 		fmt.Println("{{packageVersion}}")
```

Comment:
Might be nice to change this to "Current version: {{packageVersion}}" now that there's going to be more printed out below